example/app.js

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2014,2016. All Rights Reserved.
+// Node module: loopback-context
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+// In LB application you should require it like:
+// var LoopBackContext = require('loopback-context');
+var LoopBackContext = require('../');
+var app = loopback();
+var contextPerRequest = require('../server/middleware/per-request.js');
+var loopback = require('loopback');
+
+// Configure the context middleware
+app.middleware('initial', contextPerRequest());
+
+// Store a request property in the context
+app.use(function saveHostToContext(req, res, next) {
+  var ns = LoopBackContext.getCurrentContext();
+  if (ns)
+    ns.set('host', req.hostname);
+  next();
+});
+
+app.use(loopback.rest());
+
+var Color = loopback.createModel('color', {'name': String});
+Color.beforeRemote('**', function(ctx, unused, next) {
+  // Inside LoopBack code, you can read the property from the context
+  var ns = LoopBackContext.getCurrentContext();
+  if (ns)
+    console.log('Request to host %s', ns && ns.get('host'));
+  next();
+});
+
+app.dataSource('db', {connector: 'memory'});
+app.model(Color, {dataSource: 'db'});
+
+app.listen(3000, function() {
+  console.log('A list of colors is available at {{http://localhost:3000/colors}}');
```

Comment:
The primary noun in the sentence is `list`, not `colors`; `A list of colors is available` is correct.
`are` would match a sentence like `Lists of colors are available`.